src/main/java/org/cascadebot/cascadebot/music/CascadeLavalinkPlayer.java

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright (c) 2020 CascadeBot. All rights reserved.
+ *  Licensed under the MIT license.
+ */
+
+package org.cascadebot.cascadebot.music;
+
+import com.sedmelluq.discord.lavaplayer.filter.equalizer.Equalizer;
+import com.sedmelluq.discord.lavaplayer.track.AudioTrack;
+import lavalink.client.io.LavalinkSocket;
+import lavalink.client.io.jda.JdaLink;
+import lavalink.client.player.LavalinkPlayer;
+import lavalink.client.player.event.IPlayerEventListener;
+import lombok.Getter;
+import net.dv8tion.jda.api.entities.VoiceChannel;
+import org.cascadebot.cascadebot.CascadeBot;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class CascadeLavalinkPlayer implements CascadePlayer {
+
+    @Getter
+    private Map<Integer, Float> currentBands = new HashMap<>();
+
+    private LavalinkPlayer lavalinkPlayer;
+
+    public CascadeLavalinkPlayer(LavalinkPlayer lavalinkPlayer) {
+        for (int i = 0; i < 15; i++) {
```

Comment:
Magic number! Use band count please

Reply:
Also this has been done twice, maybe make a resetBands function?

Reply:
Thats for crating the initial local information on the bands. it doesn't actually set them server side.

Reply:
Got it